src/raftstore/store/snap.rs

```diff
@@ -646,6 +650,7 @@ impl Snap {
             // to indicate if the sst file is empty.
             if cf_file.kv_count > 0 {
                 fs::rename(&cf_file.tmp_path, &cf_file.path)?;
+                fsync_by_path(&self.dir_path)?;
```

Comment:
I don't think it's necessary. The files will be removed when restarted anyway.

Reply:
Removed